src/components/profession-hero/ProfessionHero.js

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable prettier/prettier */
+/* eslint-disable react/destructuring-assignment */
+/* eslint-disable react/prop-types */
+
+import React from 'react';
+import './ProfessionHero.css';
+
+function ProfessionHero(props) {
+  return (
+    <section className="profession">
+      <div className="profession__intro-group">
+        <img className="profession__image" src={props.professionTitle.image} alt="beep" />
```

Comment:
Don't forget to change the alt text here 😉

Reply:
:) beep

Reply:
whoops! kk ill change it